e2e/cypress/integration/login/tutorial_navigation_and_links_spec.js

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Test Tutorial Navigation', () => {
+    before(() => {
+        cy.loginAsNewUser({}, false);
+    });
+
+    it('MM-13989 - Tutorial Navigation and Links', () => {
+        // * Check that step one displays after new user signs in.
+        checkStepOne();
+
+        // # Click the second dot
+        cy.get('#tutorialIntroCircle1').click();
+
+        // * Verify the second step displays.
+        checkStepTwo();
+
+        // # Click the third dot.
+        cy.get('#tutorialIntroCircle2').click();
+
+        // * Verify the third step displays.
+        checkStepThree();
+
+        // # click the first dot.
+        cy.get('#tutorialIntroCircle0').click();
+
+        // * Verify that the user returns to the first step.
+        checkStepOne();
+
+        // # Click the Next button.
+        cy.get('#tutorialNextButton').click();
+
+        // * Verify that the second step displays.
+        checkStepTwo();
+
+        // * Test the App Download Image link to ensure it returns a 200 result.
+        cy.get('#appDownloadImage').then(($a) => {
+            const href = $a.prop('href');
+
+            cy.request(href).then((response) => {
+                expect(response.status).to.eq(200);
+            });
+        });
+
+        //* Check the Off-Topic sidebar link to ensure it is not active.
+        cy.get('#sidebarItem_off-topic').parent().should('not.have.class', 'active');
+
+        // # Click the Off-Topic sidebar link
+        cy.get('#sidebarItem_off-topic').click();
+
+        // * Verify the Off-Topic sidebar link is now active.
+        cy.get('#sidebarItem_off-topic').parent().should('have.class', 'active');
+
+        // * Verify that the second step of the tutorial still displays:
+        checkStepTwo();
+
+        // # Click the Next button
+        cy.get('#tutorialNextButton').click();
+
+        // * Verify that the third step displays.
+        checkStepThree();
+
+        // # Click the Next button.
+        cy.get('#tutorialNextButton').click();
+
+        // * Verify that the Off-Topic channel displays.
+        checkOffTopicChannel();
+
+        // # Logg the current user out.
+        cy.apiLogout();
+
+        // # Create another new user with the tutorial bypass flag set to true.
```

Comment:
Comment should say "...set to false", right?

Reply:
Corrected comment